src/index.tsx

```diff
@@ -12,7 +12,9 @@ function getValue(target: any, { isCheckbox }: { isCheckbox: boolean }) {
 
   return target
     ? isCheckbox
-      ? target.checked
+      ? isUndefined(target.checked)
+        ? target.checked
```

Comment:
@bluebill1049 Omg, I missed a point. It should be `target` not `target.checked` ... Sorry! I will create new PR...

Reply:
oops no worries! i will release another beta once it's merged in